components/Sidebar/panels/FriendItem.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from "next/link";
+
+const FriendItem = ({ track }: { track: SpotifyApi.TrackObjectFull }) => (
+  <div>
+    <div className="flex flex-wrap">
+      <h4 className="text-xs">
+        <Link
+          className="link link-hover link-secondary truncate"
+          href={track.external_urls.spotify}
+        >
+          {track.name}
+        </Link>
+      </h4>
+      <h4 className="text-xs">&nbsp;•&nbsp;</h4>
+      {track.artists.map((artist) => (
```

Comment:
These (+ ArtistLink) shouldn't be separate headers, just put one big wrapping h5